keras_cv/models/resnet_v2.py

```diff
@@ -355,6 +376,37 @@ def ResNetV2(
 
     if weights is not None:
         model.load_weights(weights)
+    # Set this private attribute for recreate backbone model with outputs at each of the
+    # resolution level.
+    model._backbone_level_outputs = stack_level_outputs
+
+    def as_backbone(self):
+        """Convert the Resnet application model into a model backbone for other tasks.
+
+        The backbone model will usually take same inputs as the original application
+        model, but produce multiple outputs, one for each feature level. Those outputs
+        can be feed to network downstream, like FPN and RPN.
+
+        The output of the backbone model will be a dict with int as key and tensor as
+        value. The int key represent the level of the feature output.
```

Comment:
Please define "level"
From "Practical ML for CV": "A typical feature pyramid has five levels corresponding to scales P3, P4, P5, P6, P7 in the backbone. Scale Pn represents a feature map 2n times smaller in width and height than the input image."
What does "level" stand for in the docstring ?